src/ReaderHTML.h

```
@@ -431,8 +443,54 @@ const char READER_HTML[] PROGMEM = R"rawliteral(
       }, 1000);
     }
 
+    var u1AssignBusy = false;
+
+    function renderU1Stage(s) {
+      var panel = document.getElementById('u1StagePanel');
+      var info = document.getElementById('u1StageInfo');
+      if (s && s.u1_staged) {
```

Clang (14.0.6)

[warning] repeated branch in conditional chain

(bugprone-branch-clone)